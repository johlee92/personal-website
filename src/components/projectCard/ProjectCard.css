.projectCard-container {
    width: 100%;
    border: 2px darkgray solid;
    max-width: 720px;
    margin-bottom: 25px;
}

.projectCard h2{
    margin: 0;
}

.projectCard-description {
    padding: 10px;
}

.projectCard-image {
    display: none;
    width: 100%;
}

.projectCard-icon {
    width: 30px;
    height: 30px;
}

button.projectCard {
    width: 120px;
    height: 30px;
    font-family: 'Noto Sans JP', sans-serif;
    text-align: center;
    font-size: 16px;
    cursor: pointer;
    border: 3px black solid;
    background-color: white;
    margin-right: 10px;
    margin-top: 5px;
}

button.projectCard:hover {
    color: white;
    background-color: darkgray;
}

@media (min-width: 720px) {
    .projectCard-image {
        display: block;
    }
}