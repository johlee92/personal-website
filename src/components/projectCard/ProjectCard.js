import React, { Component } from 'react';
import './ProjectCard.css';

class ProjectCard extends Component {
    render() {
        return (
            <div>
            </div>
        )
    }
}

export default ProjectCard;