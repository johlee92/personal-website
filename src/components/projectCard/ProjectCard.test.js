import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import ProjectCard from './ProjectCard';

describe(`ProjectCard component`, () => {
    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(
                <ProjectCard />
            , div);
        ReactDOM.unmountComponentAtNode(div);
    });

    it('renders the component as expected', () => {
        const tree = renderer
            .create(
                <ProjectCard />
            )
            .toJSON();
        expect(tree).toMatchSnapshot();  
    });
})
