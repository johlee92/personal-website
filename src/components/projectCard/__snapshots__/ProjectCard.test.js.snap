// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProjectCard component renders the component as expected 1`] = `<div />`;
